lis/surfacemodels/land/ac.7.2/ac72_setup.F90

```diff
@@ -612,29 +617,79 @@ subroutine AC72_setup()
         enddo
      endif
 
-     write(LIS_logunit,*) "[INFO] AC72: reading parameter AC_Tmin_clim from ",&
-          trim(LIS_rc%paramfile(n))
-     do k = 1, 12
-        call AC72_read_MULTILEVEL_param(n, AC72_struc(n)%LDT_ncvar_tmincli_monthly, k, placeholder)
-        do t = 1, LIS_rc%npatch(n, mtype)
-           col = LIS_surface(n, mtype)%tile(t)%col
-           row = LIS_surface(n, mtype)%tile(t)%row
-           ! Climatology is rounded to 2 decimals in AquaCrop
-           AC72_struc(n)%ac72(t)%tmincli_monthly(k) = anint(placeholder(col, row)*100)/100 - LIS_CONST_TKFRZ
+     ! Read and correct climatology
+     do m = 1, LIS_rc%nmetforc ! loop over met forcing sources (only works with overlay is m > 1)
+        if (trim(LIS_rc%metforc_blend_alg).ne."overlay") then
+           write(LIS_logunit,*) "[ERR] AC72 only runs with overlay option for the met forcing blending."
+           write(LIS_logunit,*) "[ERR] Program stopping ..."
+           call LIS_endrun
+        endif
+        write(LIS_logunit,*) "[INFO] AC72: reading parameter AC_Tmin_clim_"//trim(LIS_rc%metforc(m))//" from ",&
+                                trim(LIS_rc%paramfile(n))
+        do k = 1, 12
+           call AC72_read_MULTILEVEL_param(n, trim(AC72_struc(n)%LDT_ncvar_tmincli_monthly)//"_"//trim(LIS_rc%metforc(m)), k, placeholder)
+           do t = 1, LIS_rc%npatch(n, mtype)
+              col = LIS_surface(n, mtype)%tile(t)%col
+              row = LIS_surface(n, mtype)%tile(t)%row
+
+              if (placeholder(col, row).ne.LIS_rc%udef) then ! take only valid T for overlay
+                 ! Apply lapse-rate correction if turned on (to 2 m above surface)
+                 if (LIS_rc%met_ecor(m) == "lapse-rate") then
+                    if (LIS_rc%metforc(m) == "MERRA2") then
+                       if (merra2_struc(n)%uselml == 0) then
+                          elevdiff = (LIS_domain(n)%tile(t)%elev + 2) &
+                          - (LIS_forc(n,m)%modelelev(LIS_domain(n)%tile(t)%index) - 8)
```

Comment:
If uselml = 0, does that not mean that the elevdiff = 0? MERRA2 forcings are then diagnosed at 2 m? 
And I do not understand the "-8 m part".